intTests/test_external_abc/test.saw

```diff
@@ -16,16 +16,16 @@ let q_unsat = {{ \(x:[8]) -> x != 0 /\ x+x == x }};
 
 write_verilog "write_verilog_unsat.v" q_unsat;
 write_smtlib2_w4 "write_smtlib2_w4_unsat.smt2" q_unsat;
-write_aig_external "write_aig_external_unsat.aig" q_unsat;
-write_cnf_external "write_aig_external_unsat.cnf" q_unsat;
+write_aig_external "./write_aig_external_unsat.aig" q_unsat;
+write_cnf_external "./write_aig_external_unsat.cnf" q_unsat;
```

Comment:
Just to be clear: The `./` part is not suddenly *required* now, is it? You've just added it in this test case to exercise the new `takeBaseName` functionality, because the `./` prefix wouldn't have worked before?

Reply:
Yes, exactly. It most certainly is not required, but this approach seemed like an easy way to make sure it was working.